tests/integration/test_pip_packages.py

```diff
@@ -31,14 +31,25 @@ def test_all_pip_packages(env_name, test_env, tmpdir):
         payload={"repo": env_data["repo"], "ref": env_data["ref"], "pkg_managers": ["pip"]}
     )
     completed_response = client.wait_for_complete_request(initial_response)
+    response_data = completed_response.data
+    utils.sort_packages_in_place(response_data["dependencies"])
+    utils.sort_packages_in_place(response_data["packages"])
+
+    for idx in range(len(response_data["packages"])):
+        utils.sort_packages_in_place(response_data["packages"][idx]["dependencies"])
```

Comment:
As we add more tests, it will be nice to make another util function which does this so that we do not have to repeat the same 4 lines everywhere. If the format of expected data matched the response format more closely, we could also use that function to sort the expected data.

I'd be ok with leaving this for later, when it is needed